examples/simple/build.gradle

```diff
@@ -4,9 +4,9 @@ buildscript {
     }
     dependencies {
         classpath "org.grails:grails-gradle-plugin:$grailsVersion"
-        classpath "org.grails.plugins:hibernate5:${hibernateGormVersion}"
-        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:${assetPipelineVersion}"
-        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.0"
+        classpath "org.grails.plugins:hibernate5:$hibernateGormVersion"
+        classpath "com.bertramlabs.plugins:asset-pipeline-gradle:$assetPipelineVersion"
+        classpath "gradle.plugin.com.github.erdi.webdriver-binaries:webdriver-binaries-gradle-plugin:2.7"
```

Comment:
Can you remove the webdriver plugin and switch to `ContainerGebSpec` if Geb is used in this project?  

To remove: 

* Delete the web driver plugin
* Remove references to selenium versions & dependencies
* Remove GebConfig.groovy
* Remove geb dependencies
* Add the single dependency:

         integrationTestImplementation testFixtures(buildsrcLibs.geb.plugin)